client/landing/stepper/declarative-flow/test/site-migration-flow.tsx

```diff
@@ -41,6 +43,16 @@ describe( 'Site Migration Flow', () => {
 		const testSettingsEndpoint = '/rest/v1.4/sites/example.wordpress.com/settings';
 		nock( apiBaseUrl ).get( testSettingsEndpoint ).reply( 200, {} );
 		nock( apiBaseUrl ).post( testSettingsEndpoint ).reply( 200, {} );
+		jest
+			.spyOn( config, 'isEnabled' )
+			.mockImplementation( ( key ) =>
+				key === 'migration-flow/remove-processing-step' ? false : originalIsEnabled( key )
+			);
```

Comment:
It might be worth having the default be a passthrough, and then mock the specific value explicitly in the two tests where a specific value is expected, possibly using `when` from `jest-when`. (Non-blocking.)
```suggestion
		// Set up a spy on config.isEnabled() so we can mock it in specific tests
		jest
			.spyOn( config, 'isEnabled' )
			.mockImplementation( ( key ) =>
				originalIsEnabled( key )
			);
```

We'd then have the following later:
`when( config.isEnabled as jest.Mock ).calledWith( 'migration-flow/remove-processing-step' ).mockReturnValue( true ); // or false, when needed `

Reply:
we will need to remove all the new screen is is enabled and stable on production. So we can adjust then.